programs/compressed-pda/src/compressed_account.rs

```diff
@@ -4,23 +4,27 @@ use anchor_lang::prelude::*;
 use light_hasher::{Hasher, Poseidon};
 use light_utils::hash_to_bn254_field_size_be;
 
+#[derive(Debug, PartialEq, Default, Clone, AnchorSerialize, AnchorDeserialize)]
```

Comment:
one general thought i had - are we using all the derived traits for all structs?

Reply:
in most cases yes